### PR TITLE
remote_file: remove mention to verifications property

### PR DESCRIPTION
**Pull request by tas50**
This is not a thing

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

**Review by kagarmoe — APPROVED** [review]

File: data/infra/resources/remote_file.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ resource_description_list:
 - markdown: Use the **remote_file** resource to transfer a file from a remote location
     using file specificity. This resource is similar to the **file** resource.
 - note:
-    markdown: Fetching files from the `files/` directory in a cookbook should be done
+  - markdown: Fetching files from the `files/` directory in a cookbook should be done
```
Comment (IanMadd) [inline data/infra/resources/remote_file.yaml]: The note text is missing.
```suggestion
  markdown: Fetching files from the `files/` directory in a cookbook should be done
```

Comment (tas50) [reply data/infra/resources/remote_file.yaml]: This is how we're generating this one currently. Should this just be a single hash?

Comment (IanMadd) [reply data/infra/resources/remote_file.yaml]: Yep. There's a few we need to fix.

File: data/infra/resources/remote_file.yaml [hunk 2]
```diff
@@ -9,7 +9,7 @@ resource_description_list:
 - markdown: Use the **remote_file** resource to transfer a file from a remote location
     using file specificity. This resource is similar to the **file** resource.
 - note:
-    markdown: Fetching files from the `files/` directory in a cookbook should be done
+  markdown: Fetching files from the `files/` directory in a cookbook should be done
```
Comment (IanMadd) [inline data/infra/resources/remote_file.yaml]: This time my suggestion should be right.
```suggestion
    markdown: Fetching files from the `files/` directory in a cookbook should be done
```

**Comment by tas50** [comment]
Rebased to fix the DCO

**Pull request merged**